addon/-private/system/relationships/belongs-to.js

```diff
@@ -68,6 +68,31 @@ import { DEBUG } from '@glimmer/env';
 
   will lookup for a Post type.
 
+  #### Sync relationships
+
+  Ember Data resolves sync relationships with the related resources
+  available in its local store, hence it is expected these resources
+  to be loaded before or along-side the primary resource.
+
+  ```app/models/comment.js
+  import DS from 'ember-data';
+
+  export default DS.Model.extend({
+    post: DS.belongsTo('post', {
+      async: false
+    })
+  });
+  ```
+
+  In contrast to async relationship, accessing a sync relationship
+  will never trigger a request to fetch the resource,
+  and it will always return the existing local resource.
```

Comment:
let's be more specific: `it will always return the record (Model instance) for the existing local resource, or null`